README.md

```diff
@@ -249,6 +249,35 @@ If user (with registered identifier using `registerPerference()`) opts out from
 
 Campaign impressions (displays) are counted locally for each user. Meaning that a campaign with maxImpression value of 3 will be displayed to each user (registered with `registerPerference()`) max 3 times. Campaign's max impression number can be modified in the dashboard/backend. Then the SDK, after next ping call, will compare new value with old max impression number and add the difference to the current impression counter. The max impression data is not shared between devices. The same applies for anonymous user.
 
+## (Optional) How enable custom fonts in your Campaigns
+
+The SDK will optionally use custom fonts in your Campaigns if your app has them pre-registered. Specify one for the text body/headers and one for the buttons in either `ttf` or `otf` format. Fallsback to the system font if unset.
+
+First add the two font files to your Xcode target.
+
+Get the "PostScript name" of your fonts by:
+
+```bash
+$ fc-scan customfont-medium.otf --format "%{postscriptname}\n"
+
+AdventPro-Medium
+```
+
+In your `Info.plist` configuration, set the PostScript names under `InAppMessagingCustomFontNameText` and `InAppMessagingCustomFontNameButton` along with the file names of both fonts under `UIAppFonts`.
+
```

Comment:
Do they need to be in the plist? Could we just pass them in as optional params to `configure()`?

Reply:
We could. But I put it in the plist since the config URL is also in the plist too.